package.json

```diff
@@ -13,21 +13,20 @@
   },
   "scripts": {
     "prepare": "husky install",
-    "clean": "rm -rf ./build ./dist",
+    "clean": "rm -rf ./build",
     "lint": "eslint ./src",
     "format": "prettier -w ./src config-overrides.js",
     "tsc": "tsc",
-    "build": "DOTENV_CONFIG_PATH=.env.production GENERATE_SOURCEMAP=false react-app-rewired build",
-    "build:dev": "GENERATE_SOURCEMAP=false react-app-rewired build",
-    "watch": "node ./scripts/watch",
-    "start": "react-app-rewired start",
-    "test": "react-app-rewired test",
+    "build": "tsc && NODE_ENV=production webpack",
+    "build:dev": "webpack",
+    "watch": "webpack -w",
+    "watch:prod": "DOTENV_CONFIG_PATH=.env.production webpack -w",
+    "test": "jest",
     "stats": "webpack-bundle-analyzer build/stats.json",
```

Comment:
Do we still have this feature available? I see you have removed the `BUNDLE_STATS` env var

Reply:
Yes we have it. I've removed BUNDLE_STATS from the .env files but you can still pass it as an environment variable and webpack will generate the stats like before. I just thought it could be more convenient to add it on the command line on the rare occasions when you need it than to edit your .env file.